airflow/models/baseoperator.py

```diff
@@ -377,9 +377,10 @@ def __init__(
                 self
             )
         self._schedule_interval = schedule_interval
-        self.retries = retries
+        self.retries = retries if retries is not None else \
+            int(configuration.conf.get('core', 'default_task_retries', fallback=0))
         self.queue = queue
-        self.pool = pool
+        self.pool = Pool.DEFAULT_POOL_NAME if pool is None else pool
```

Comment:
This seems relevant to the above comment and change.

Reply:
This is done as a part of https://github.com/apache/airflow/pull/8587